Frontend/ui-library/src/Config/ConfigUI.ts

```diff
@@ -126,148 +128,160 @@ export class ConfigUI {
+                this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.MouseInput));
 
-        if (isSettingEnabled(settingsConfig, Flags.GamepadInput))
-            this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.GamepadInput));
+            if (isSettingEnabled(settingsConfig, Flags.FakeMouseWithTouches))
+                this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.FakeMouseWithTouches));
 
-        if (isSettingEnabled(settingsConfig, Flags.XRControllerInput))
-            this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.XRControllerInput));
+            if (isSettingEnabled(settingsConfig, Flags.TouchInput))
+                this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.TouchInput));
 
-        /* Setup all encoder related settings under this section */
-        const encoderSettingsSection = this.buildSectionWithHeading(settingsElem, 'Encoder');
+            if (isSettingEnabled(settingsConfig, Flags.GamepadInput))
+                this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.GamepadInput));
 
-        if (isSettingEnabled(settingsConfig, NumericParameters.CompatQualityMin))
-            this.addSettingNumeric(
-                encoderSettingsSection,
-                this.numericParametersUi.get(NumericParameters.CompatQualityMin)
-            );
-        if (isSettingEnabled(settingsConfig, NumericParameters.CompatQualityMax))
-            this.addSettingNumeric(
-                encoderSettingsSection,
-                this.numericParametersUi.get(NumericParameters.CompatQualityMax)
-            );
-
-        const preferredCodecOption = this.optionParametersUi.get(OptionParameters.PreferredCodec);
-        if (isSettingEnabled(settingsConfig, OptionParameters.PreferredCodec))
-            this.addSettingOption(
-                encoderSettingsSection,
-                this.optionParametersUi.get(OptionParameters.PreferredCodec)
-            );
-        if (
-            preferredCodecOption &&
-            [...preferredCodecOption.selector.options]
-                .map((o) => o.value)
-                .includes('Only available on Chrome')
-        ) {
-            preferredCodecOption.disable();
+            if (isSettingEnabled(settingsConfig, Flags.XRControllerInput))
+                this.addSettingFlag(inputSettingsSection, this.flagsUi.get(Flags.XRControllerInput));
         }
 
-        if (isSettingEnabled(settingsConfig, OptionParameters.PreferredQuality))
-            this.addSettingOption(
-                encoderSettingsSection,
-                this.optionParametersUi.get(OptionParameters.PreferredQuality)
-            );
+        if (isSectionEnabled(settingsConfig, Sections.Encoder)) {
+            /* Setup all encoder related settings under this section */
+            const encoderSettingsSection = this.buildSectionWithHeading(settingsElem, 'Encoder');
```

Comment:
Perhaps heading should also come from the new `Sections` type instead of hardcoded string?

Reply:
Yep, forgot to do that. Fixed!